docs/docs/gatsby-cli.md

```diff
@@ -3,17 +3,17 @@ title: Commands (Gatsby CLI)
 tableOfContentsDepth: 2
 ---
 
-The Gatsby command line tool (CLI) is the main entry point for getting up and running with a Gatsby application and for using functionality including like running a development server and building out your Gatsby application for deployment.
+Gatsby কমান্ড লাইন টুল (CLI) হল Gatsby অ্যাপ্লিকেশন শুরু করার প্রথম ধাপ, যা দিয়ে ডেভেলপমেন্ট সার্ভার চালানো এবং ডিপ্লয়মেন্টের জন্য Gatsby অ্যাপ্লিকেশন তৈরি করাসহ বিভিন্ন কাজ করা যায়।
 
-_We provide similar documentation available with the gatsby-cli [README](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-cli/README.md), and our [cheat sheet](/docs/cheat-sheet/) has all the top CLI commands ready to print out._
+_আমরা gatsby-cli [README](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-cli/README.md) এর সাথে অনুরূপ ডকুমেন্টেশন সরবরাহ করি, এবং আমাদের [চিটশীট](/docs/cheat-sheet)-এ সমস্ত শীর্ষ CLI কমান্ড মুদ্রণের জন্য প্রস্তুত রয়েছে।_
 
-## How to use gatsby-cli
+## কীভাবে gatsby-cli ব্যবহার করবেন
 
-The Gatsby CLI (`gatsby-cli`) is packaged as an executable that can be used globally. The Gatsby CLI is available via [npm](https://www.npmjs.com/) and should be installed globally by running `npm install -g gatsby-cli` to use it locally.
+Gatsby CLI-টি (`gatsby-cli`) প্যাকেজ করা হয়েছে একটি এক্সেকিউটাবল হিসাবে যেটি গ্লোবালি ব্যবহার করা যেতে পারে। Gatsby CLI [npm](https://www.npmjs.com/) -এ পাওয়া যায় এবং এটিকে গ্লোবালি ইনস্টল করে লোকালি ব্যবহার করা যেতে পারে এই কম্যান্ড দ্বারা `npm install -g gatsby-cli`.
 
-Run `gatsby --help` for full help.
+সম্পূর্ণ সহায়তার জন্য `gatsby --help` চালান।
 
-You can also use the `package.json` script variant of these commands, typically exposed _for you_ with most [starters](/docs/starters/). For example, if you want to make the [`gatsby develop`](#develop) command available in your application, open up `package.json` and add a script like so:
+আপনি `package.json` এর স্ক্রিপ্ট দিয়ে এই কমান্ডগুলি ব্যবহার করতে পারেন, সাধারণত বেশিরভাগ [স্টার্টার্সে](/docs/starters/)  _আপনার জন্য_ প্রকাশ করা হয়েছে । উদাহরণস্বরূপ, আপনি যদি নিজের অ্যাপ্লিকেশনে [`gatsby develop`](#develop) কমান্ডটি ব্যবহার করতে চান তবে `package.json` খুলুন এবং এই রকম স্ক্রিপ্ট যুক্ত করুন ঃ
```

Comment:
```suggestion
আপনি `package.json` এর স্ক্রিপ্ট দিয়ে এই কমান্ডগুলি ব্যবহার করতে পারেন, সাধারণত বেশিরভাগ [স্টার্টার্সে](/docs/starters/)  _আপনার জন্য_ প্রকাশ করা হয়েছে । উদাহরণস্বরূপ, আপনি যদি নিজের অ্যাপ্লিকেশনে [`gatsby develop`](#develop) কমান্ডটি ব্যবহার করতে চান তবে `package.json` খুলুন এবং এই রকম স্ক্রিপ্ট যুক্ত করুনঃ
```

Reply:
Fix the other ones like this please.

Reply:
Interestingly they are not showing up in VS code.
![image](https://user-images.githubusercontent.com/29725587/73294016-faae2d80-422a-11ea-9bef-19dc19ad3e9d.png)

Reply:
I will try to figure out a way to fix this.

Reply:
@shakib609 I am not able to fix the issue with the `বিসর্গ`s. I would be very thankful if you can make a commit in this PR fixing them.

Reply:
Can you give a last try with this:
https://avro.im/

Reply:
Done! Please check. I wasn't aware of such an awesome tool. :sweat_smile: